cql/ManageSpecialPopulation.cql

```diff
@@ -721,7 +722,20 @@ define RecommendationForManagingPatientsAfterHysterectomy:
         date: Collate.DateOfMostRecentReport + 3 years,
         group: 'Hysterectomy (K.4)',
         details: {
-          'HPV-based testing (cotest or primary hrHPV test) is recommended every 3 years for 25 years.'
+          'HPV-based testing (cotest or primary hrHPV test) is recommended every 3 years for 25 years after hysterectomy is performed for treatment of histologic HSIL (CIN 2 or CIN 3).'
+        }
+      }
+    when ( // no hysterectomy date
+      SAR.AbsenceOrRemovalOfCervix
+    ) then 
+      {
+        short: 'Primary HPV or Cotest',
```

Comment:
```suggestion
        short: 'See Details',
```

I think this makes more sense now, since we are recommending several different things based upon the interpreted clinical scenario. "See Details" is what we write in other sections where multiple recommendations are given.